master/buildbot/newsfragments/ec2_latent_worker.bugfix

```diff
@@ -0,0 +1,3 @@
+_wait_for_request() would fail to format a log statement due to an invalid type being passed to log.msg (resulting in a broken build)
```

Comment:
Those sentence will be placed in the middle of the release notes. Please be more user oriented.

    bug fixes in EC2LatentWorker for spot request support.
    Tag support in EC2LatentWorker